duckduckgo/Android
# Update autofill to 1.0.2

Task/Issue URL: https://app.asana.com/0/1198964220583541/1200378512788370
Tech Design URL: 
CC: 

**Description**:
Updates autofill to 1.0.2

**Steps to test this PR**:
1. Checkout branch
1. Run `git submodule`
2. Confirm autofill commit hash is `8412aa5d134b3d79f13eb0b2cdd87048fe6cb2ba`


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
